docs/viewports/how-viewports-work.md

```diff
@@ -0,0 +1,383 @@
+
+**`toggleViewport(viewport)`**—Assigns CSS classes to the `viewportClasses` observable, based on the selected viewport name passed in as an argument (`viewport`). The CSS classes in `viewportClasses` set the stage width. This function also triggers the `stage:viewportChangeAfter` events so that content types can handle these events and make changes to their own layouts. For more details, see [Stage events](#stage-events) at the end of this topic.
+
+## `view.xml`
+
+Page Builder's `view.xml` file provides the default configuration data for the `desktop` and `mobile` viewports, as shown here:
+
+```xml
+<vars module="Magento_PageBuilder">
+    <var name="breakpoints">
+        <var name="desktop">
+            <var name="label">Desktop</var>
+            <var name="stage">true</var>
+            <var name="default">true</var>
+            <var name="class">desktop-switcher</var>
+            <var name="icon">Magento_PageBuilder::css/images/switcher/switcher-desktop.svg</var>
+            <var name="conditions">
+                <var name="min-width">1024px</var>
+            </var>
+            <var name="options">
+                <var name="products">
+                    <var name="default">
+                        <var name="slidesToShow">5</var>
+                    </var>
+                </var>
+            </var>
+        </var>
+        <var name="mobile">
+            <var name="label">Mobile</var>
+            <var name="stage">true</var>
+            <var name="default">false</var>
+            <var name="class">mobile-switcher</var>
+            <var name="icon">Magento_PageBuilder::css/images/switcher/switcher-mobile.svg</var>
+            <var name="media">only screen and (max-width: 768px)</var>
+            <var name="conditions">
+                <var name="max-width">768px</var>
+                <var name="min-width">640px</var>
+            </var>
+            <var name="options">
+                <var name="products">
+                    <var name="default">
+                        <var name="slidesToShow">3</var>
+                    </var>
+                </var>
+            </var>
+        </var>
+    </var>
+</var>
+```
+
+Configuration data descriptions:
+
+- `Magento_PageBuilder` - Defines the module to which viewport data applies.
+- `breakpoints` - Defines `viewports` object for the module.
+- `desktop` and `mobile` - Defines viewport objects named `desktop` and `mobile`.
+- `label` - (string) Defines title of tooltip.
+- `stage` - (bool) ????
+- `default` - (bool) Determines if the viewport is selected when the stage is loaded.
+- `class` - (string) CSS class to apply to the viewport switcher button (not the stage).
+- `icon` - (string) - URL to the button image (SVG) location.
+- `media` - (string) ????
```

Comment:
media query rule that used in the admin area

Reply:
Also what I thought, but couldn't see differences when I changed it in my example. Again, probably something happened in my example.